scripts/pages.mjs

```diff
@@ -43,6 +43,31 @@ class ClassPages extends Application {
     html[0].querySelectorAll(".class-nav img[data-action]").forEach(n => {
       n.addEventListener("click", this._onClickClass.bind(this));
     });
+    html[0].querySelector(".class-nav .gallery").addEventListener(
+      "wheel", debounce((event) => this._onGalleryScroll(event, html[0].querySelector(".class-nav .gallery")), 50)
+    );
+  }
+
+  /**
+   * Handle the scroll event for the class icons gallery.
+   * @param {WheelEvent} event        The scroll event.
+   * @param {HTMLElement} gallery     The gallery element containing the class icons.
+   */
+  _onGalleryScroll(event, gallery) {
+    const direction = Math.sign(event.deltaY);
+    const activeIcon = gallery.querySelector("img.active");
+    let nextIcon;
+
+    if (direction > 0) {
+      nextIcon = activeIcon.nextElementSibling || gallery.querySelector("img:first-child");
+    } else {
+      nextIcon = activeIcon.previousElementSibling || gallery.querySelector("img:last-child");
+    }
+
+    if (nextIcon) {
+      nextIcon.scrollIntoView({ behavior: "smooth", inline: "center" });
+      this._onClickClass({ currentTarget: nextIcon });
```

Comment:
It may be worthwhile to create a standard function in a follow-up PR that renders on a specific page, but for now I think calling `render` with the correct arguments is much cleaner than providing this 'dummy event' object to mimic the behavior.

Reply:
I tried that.